frontend/javascripts/oxalis/model/sagas/volumetracing_saga.tsx

```diff
@@ -782,6 +787,26 @@ function* updateHoveredSegmentId(): Saga<void> {
   }
 }
 
+export function* updateClickedSegments(
+  action: ClickSegmentAction | SetActiveCellAction,
+): Saga<void> {
+  // If one or zero segments are selected, update selected segments in store
+  // Otherwise, the multiselection is kept.
+  const { segmentId } = action;
+  let layerName: string | undefined;
+  const segmentationLayer = yield* call([Model, Model.getVisibleSegmentationLayer]);
+  layerName = segmentationLayer?.name;
+  if (layerName == null) return;
+  const clickedSegmentId = segmentId;
+  const selectedSegmentsOrGroup = yield* select(
+    (state) => state.localSegmentationData[layerName as string]?.selectedIds,
```

Comment:
is `as string` really necessary? i wouldn't think so because of the == null check above.

Reply:
If I omit it, the typechecker warns that `Type 'undefined' cannot be used as an index type.`

Reply:
I could also use `layerName!`